handbook/engineering/search/iteration_log.md

```diff
@@ -14,9 +14,41 @@ This document contains the goals and work log for the search team's [2-week iter
 
 ## 2020-09-21 to 2020-10-02
 
-### $GOAL_OR_THEME
+### Scale indexed search to 500k repositories
 
-- **Owner(s):** $OWNER
+- **Owner(s):** Stefan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Streaming search
+
+- **Owner(s):** Keegan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Enterprise homepage
+
+- **Owner(s):** Juliana
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Search tour
+
+- **Owner(s):** Farhan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
```

Comment:
@attfarhan can you document the desired outcomes for the iteration on the search tour?

Reply:
```suggestion
    - The [feedback from the first round of user testing](https://github.com/sourcegraph/sourcegraph/issues/13944) has been implemented.
    - The design team is able to run subsequent rounds of user testing for feedback.
    - If user testing is completed, and feedback is addressed, the tour is turned on for all Sourcegraph.com new users by default.
```